packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -216,6 +220,85 @@ export interface ClusterOptions {
    * @default true
    */
   readonly outputConfigCommand?: boolean;
+
+  /**
+   * Configure access to the Kubernetes API server endpoint.
+   * This feature is only available for kubectl enabled clusters, i.e `kubectlEnabled: true`.
+   *
+   * @see https://docs.aws.amazon.com/eks/latest/userguide/cluster-endpoint.html
+   *
+   * @default - Private and Public. The cluster endpoint is accessible from outside of your VPC.
```

Comment:
Use the actual value instead of `-` if possible

Reply:
Even though the actual value is complex and doesn't directly explain the behavior? It would require another hop from the user to look at the docstring of `EndpointAccess.privateAndPublic()`.

Reply:
`EndpointAccess.privateAndPublic()` seems like a reasonable value to put there

Reply:
👍